README.md

```diff
@@ -17,37 +17,37 @@ The general edition of the Guidebook is managed by [@w3c/guidebook](https://gith
 
 ## In this repository
 
-* [/chair/](https://github.com/w3c/Guide/tree/main/chair)
+* [/chair/](tree/main/chair)
   * Instructions on how to chair WGs and IGs including managing meetings, setting up the homepage, maintaining the calendar.
-* [/council/](https://github.com/w3c/Guide/tree/main/council)
+* [/council/](tree/main/council)
   * Anything related to councils.
-* [/documentreview/](https://github.com/w3c/Guide/tree/main/documentreview)
+* [/documentreview/](tree/main/documentreview)
   * How to do wide review.
-* [/editor/](https://github.com/w3c/Guide/tree/main/editor)
+* [/editor/](tree/main/editor)
   * How to author a specification, including ReSpec or other spec making programs.
-* [/manual-of-style/](https://github.com/w3c/Guide/tree/main/manual-of-style)
+* [/manual-of-style/](tree/main/manual-of-style)
   * The W3C Manual of Style for editors.
-* [/meetings/](https://github.com/w3c/Guide/tree/main/meetings)
+* [/meetings/](tree/main/meetings)
   * Anything related to organizing meetings, including events.
-* [/process/](https://github.com/w3c/Guide/tree/main/process)
+* [/process/](tree/main/process)
   * Anything related to additional guidelines when following the process.
   * You may involve [@w3c/w3process-editors](https://github.com/orgs/w3c/teams/w3process-editors).
-* [/process/tilt/](https://github.com/w3c/Guide/tree/main/process/tilt)
+* [/process/tilt/](tree/main/process/tilt)
   * Anything related to technical Team decision and verification, except for transitions.
   * Managed by [@w3c/tilt](https://github.com/orgs/w3c/teams/tilt)
-* [/standards-track/](https://github.com/w3c/Guide/tree/main/standards-track).
+* [/standards-track/](tree/main/standards-track).
   * Best Practices for Bringing Work to the W3C Recommendation Track.
-* [/teamcontact/](https://github.com/w3c/Guide/tree/main/teamcontact)
+* [/teamcontact/](tree/main/teamcontact)
   * Guidance on the role of the team contact, and what to expect from them.
-* [/transitions/](https://github.com/w3c/Guide/tree/main/transitions)
+* [/transitions/](tree/main/transitions)
   * Organize a Technical Report Transition.
   * Managed by [@w3c/transitions](https://github.com/orgs/w3c/teams/transitions).
-* [/other/](https://github.com/w3c/Guide/tree/main/other)
+* [/other/](tree/main/other)
   * Related to [permanent groups](https://www.w3.org/groups/).
 
 # Related repositories
 * [//w3c.github.io](https://github.com/w3c/w3c.github.io)
   * How to use Git and GitHub at W3C.
-  * This repository is not exposed through [the Guidebook](https://www.w3.org/Guide/).
+  * This repository is not exposed through [the Guidebook](https://www.w3.org/guide/).
 
 Note: now that we have a repository for the guidebook, simply add your documentation into it rather than creating new repositories.
```

Comment:
```suggestion
Note: now that we have a repository for the guidebook, simply add your documentation to it, rather than creating new repositories.
```